CONTRIBUTING.md

```diff
@@ -49,21 +49,6 @@ npm start "contributors.add <YOUR_GITHUB_USERNAME>"
 Follow the prompt. If you've already added yourself to the list and are making a new type of contribution, you can run
 it again and select the added contribution type.
 
-## Committing and Pushing changes
```

Comment:
Now that I look at this, it doesn't make a lot of sense. Will close and file separate ones.